android/src/main/java/com/rncamerakit/CodeFormat.kt

```diff
@@ -0,0 +1,56 @@
+package com.rncamerakit
+
+import com.google.mlkit.vision.barcode.common.Barcode
+
+enum class CodeFormat(val code: String) {
+    CODE_128("code-128"),
+    CODE_39("code-39"),
+    CODE_93("code-93"),
+    CODABAR("codabar"),
+    EAN_13("ean-13"),
+    EAN_8("ean-8"),
+    ITF("itf"),
+    UPC_E("upc-e"),
+    QR("qr"),
+    PDF_417("pdf-417"),
+    AZTEC("aztec"),
+    DATA_MATRIX("data-matrix"),
+    UNKNOWN("unknown");
+
+    fun toBarcodeType(): Int {
+        return when (this) {
+            CODE_128 -> Barcode.FORMAT_CODE_128
+            CODE_39 -> Barcode.FORMAT_CODE_39
+            CODE_93 -> Barcode.FORMAT_CODE_93
+            CODABAR -> Barcode.FORMAT_CODABAR
+            EAN_13 -> Barcode.FORMAT_EAN_13
+            EAN_8 -> Barcode.FORMAT_EAN_8
+            ITF -> Barcode.FORMAT_ITF
+            UPC_E -> Barcode.FORMAT_UPC_E
+            QR -> Barcode.FORMAT_QR_CODE
+            PDF_417 -> Barcode.FORMAT_PDF417
+            AZTEC -> Barcode.FORMAT_AZTEC
+            DATA_MATRIX -> Barcode.FORMAT_DATA_MATRIX
+            UNKNOWN -> -1 // Or any other default value you prefer
+        }
+    }
+
+    companion object {
+        fun fromBarcodeType(barcodeType: Int): CodeFormat =
```

Comment:
We can add this annotation to enforce what kind of `Int` we are receiving here: `@Barcode.BarcodeFormat`

```suggestion
        fun fromBarcodeType(@Barcode.BarcodeFormat barcodeType: Int): CodeFormat =
```

It might work as well on the return type of `toBarcodeType`